_includes/social_links/dribbble.html

```diff
@@ -0,0 +1 @@
+<a href="https://dribbble.com/{{ include.username || site.dribbble_username }}"><svg class="svg-icon" xmlns="http://www.w3.org/2000/svg"><use xlink:href="/assets/icons.svg#dribbble"></use></svg> <span class="username">{{ site.dribbble_username }}</span></a>
```

Comment:
Apparently, this is because the *About* page does the following:

```
You can find the source code for the Jekyll new theme at:
{% include social_links/github.html username="jekyll" %} /
[minima](https://github.com/jekyll/minima)

You can find the source code for Jekyll at
{% include social_links/github.html username="jekyll" %} /
[jekyll](https://github.com/jekyll/jekyll)
```

This is why the social templates were previously separate files I guess.